pkg/cloudprovider/vsphereparavirtual/cloud.go

```diff
@@ -167,7 +167,7 @@ func (cp *VSphereParavirtual) Routes() (cloudprovider.Routes, bool) {
 
 // ProviderName returns the cloud provider ID.
 func (cp *VSphereParavirtual) ProviderName() string {
-	return ProviderName
+	return "vsphere"
```

Comment:
I do think this method warrants a comment now, for why it's vsphere and not vsphere-paravirtual here

Reply:
Cool, let me add one